cluster/charts/rook-resources/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: rook-resources
```

Comment:
what if the chart name matched the top-level crd name? For example: `rook-cluster`

Reply:
ya, `rook-cluster` sounds better. I will update this.